### PR TITLE
[script] Fix support version

### PR DESCRIPTION
**Pull request by sung-su**
- Change support dotnet sdk version from only 6 to 6 or later.

### CODE REVIEW

File: workload/scripts/workload-install.ps1 [hunk 1]
```diff
@@ -146,9 +146,9 @@ if (Get-Command $DotnetCommand -ErrorAction SilentlyContinue)
     $DotnetVersion = Invoke-Expression "& '$DotnetCommand' --version"
     $VersionSplitSymbol = '.'
     $SplitVersion = $DotnetVersion.Split($VersionSplitSymbol);
-    if ($SplitVersion[0] -ne $SupportedDotnetVersion)
+    if ($SplitVersion[0] -lt $SupportedDotnetVersion)
     {
-        Write-Host "Current .NET version is $DotnetVersion. .NET 6.0 SDK is required."
+        Write-Host "Current .NET version is $DotnetVersion. .NET SDK version 6.0 or later is required."
```
Comment (WonyoungChoi) [inline workload/scripts/workload-install.ps1:151]: Is it ok not to check minor version? If you want to check the minor version, 

```powershell
$SupportedDotnetVersion = [Version]"6.0"
....
$CurrentDotnetVersion = [Version]"$($SplitVersion[0]).$($SplitVersion[1])"
if ($CurrentDotnetVersion -lt $SupportedDotnetVersion)
{
    Write-Host "Current .NET version is $DotnetVersion. .NET SDK version $SupportedDotnetVersion or later is required."
}
```

File: workload/scripts/workload-install.ps1 [hunk 2]
```diff
@@ -146,9 +146,9 @@ if (Get-Command $DotnetCommand -ErrorAction SilentlyContinue)
     $DotnetVersion = Invoke-Expression "& '$DotnetCommand' --version"
     $VersionSplitSymbol = '.'
     $SplitVersion = $DotnetVersion.Split($VersionSplitSymbol);
```
Comment (WonyoungChoi) [inline workload/scripts/workload-install.ps1:148]: unnecessary semi-colon. 
```suggestion
    $SplitVersion = $DotnetVersion.Split($VersionSplitSymbol)
```

Comment (sung-su) [reply workload/scripts/workload-install.ps1]: fixed

File: workload/scripts/workload-install.ps1 (hunk 1 above)
Comment (sung-su) [reply workload/scripts/workload-install.ps1]: fixed

File: workload/scripts/workload-install.ps1 (hunk 2 above)
Comment (sung-su) [reply workload/scripts/workload-install.ps1]: fixed

**Review by JoonghyunCho — APPROVED** [review]
Thanks for update

**Pull request merged**